File: messages/src/main/java/com/thundashop/core/arx/Person.java

package com.thundashop.core.arx;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Person {
    public String id;
    public String lastName = "";
    public String firstName = "";
    public List<AccessCategory> accessCategories = new ArrayList();
    public Date endDate = null;
    public List<Card> cards = new ArrayList();
    public boolean deleted = false;
}

File: html5apps/terminal/events/core/arx/Person.php
<?php
class core_arx_Person {
	/** @var String */
	public $id;

	/** @var String */
	public $lastName;

	/** @var String */
	public $firstName;

	/** @var core_arx_AccessCategory[] */
	public $accessCategories;

	/** @var String */
	public $endDate;

	/** @var core_arx_Card[] */
	public $cards;

	/** @var String */
	public $deleted;

}
?>